docs/api/covidcast_signals.md

```diff
@@ -0,0 +1,140 @@
+held by CMU and are shareable with other health researchers under a data use
+agreement. No individual survey responses are shared back to Facebook.
+
+| Signal | Description |
+| --- | --- |
+| `raw_cli` | Estimated fraction of people with COVID-like illness, with no smoothing or survey weighting |
+| `raw_ili` | Estimated fraction of people with influenza-like illness, with no smoothing or survey weighting |
+| `raw_wcli` | Estimated fraction of people with COVID-like illness; adjusted using survey weights |
+| `raw_wili` | Estimated fraction of people with influenza-like illness; adjusted using survey weights |
+
+The survey weights, provided by Facebook, are intended to make the sample
+representative of the US population, according to the state, age, and gender of
+the US population from the 2018 Census March Supplement.
+
+Along with the `raw_` signals, there are additional signals with names beginning
+with `smoothed_`. These estimate the same quantities as the above signals, but
+smoothed in time to reduce day-to-day sampling noise.
+
+### `google-survey`
+
+This indicator estimates the percentage of people who know someone in their
+community with a COVID-like illness (fever, along with cough, or shortness of
+breath, or difficulty breathing). The data is based on Google-run symptom
+surveys, through publisher websites, their Opinions Reward app, and similar
+applications. Respondents can opt to skip the survey and complete a different
+one if they prefer not to answer.
+
+The surveys are just one question long, and ask "Do you know someone in your
+community who is sick (fever, along with cough, or shortness of breath, or
+difficulty breathing) right now?" Using this survey data, we estimate the
+percentage of people in a given location, on a given day, that know somebody who
+has a COVID-like illness. Note that this is tracking a different quantity than
+the surveys through Facebook, and (unsurprisingly) the estimates here tend to be
+much larger.
+
+| Signal | Description |
+| --- | --- |
+| `raw_cli` | Estimated fraction of people who know someone in their community with COVID-like illness |
+| `smoothed_cli` | Estimated fraction of people who know someone in their community with COVID-like illness, smoothed in time |
+
+### `ght`
+
+Data signal based on Google searches, provided to us by Google Health
+Trends.  Using this search data, we estimate the volume of COVID-related
+searches in a given location, on a given day.  This signal is measured in
+arbitrary units (its scale is meaningless); larger numbers represent higher
+numbers of COVID-related searches.
+
+| Signal | Description |
+| --- | --- |
+| `raw_search` | Google search volume for COVID-related searches, in arbitrary units; normalized by population |
+| `smoothed_search` | Google search volume for COVID-related searches, in arbitrary units and normalized by population, smoothed in time using a Gaussian linear smoother |
```

Comment:
```suggestion
| `smoothed_search` | Google search volume for COVID-related searches, in arbitrary units that are normalized for population, smoothed in time using a Gaussian linear smoother |
```
Clarify that arbitrary units are already normalized, rather than having arbitrary units that are then normalized

Reply:
Clarify that arbitrary units are already normalized, rather than having arbitrary units that are then normalized